### DISCUSSION TITLE
Show VLAN in the topology

### DISCUSSION THREAD

**Discussion by JessicaRigol** [netbox-community/netbox#16863] [Q&A]
Is there a way to make a VLAN appear in the topology along with the interfaces when you hover over the cable? 
For example, within this display?

![image](https://github.com/netbox-community/netbox/assets/112140945/804b3a77-b5e4-4b9c-8fb5-fb90bf4d1948)

**Comment by bogdancordos** [comment]
Hi, I think this is a discussion that you might want to open on the plugin's repo (`https://github.com/netbox-community/netbox-topology-view`s). The topology view is not a core NetBox feature.

**Reply by JessicaRigol** [reply]
Ok, thanks!
